src/XMLexport.cpp

```diff
@@ -750,31 +750,41 @@ bool XMLexport::exportTrigger(QIODevice* device)
 
 bool XMLexport::exportToClipboard(TTrigger* pT)
 {
-    setAutoFormatting(true);
-
-    QClipboard* cb = QApplication::clipboard();
-
-    QByteArray ba = "";
-    QBuffer xmlBuffer(&ba);
+    // The use of pT is a cludge - it was already used in the previously invoked
+    // in this XMLexport instance's constructor (and stored in mpTrigger) and it
+    // is only used here for its signature.
+    Q_UNUSED(pT);
```

Comment:
Let's just delete the argument then instead of declaring it and saying it's unused!

Reply:
We can't - we need it to give a different signature for each of the different `(bool) XMLexport::exportToClipboard(`....`)` methods.  That is what I am moaning about - we call a different `XMLexport` constructor with the item to export which stores a pointer to that item and sets ONE of a set of `bool` to record the type - which they could all be replaced by a single `enum` very similar to the recently revised cmViewXXX thingummyjig, then we call an `exportToClipboard` method for the particular item type with the same item pointer.  I feel we ought to be able to do something which only needs one call to the class but I am not exactly sure how {mayhaps something with a type `enum` and a `QVariant`}...